queries/highlights.scm

```diff
@@ -10,9 +10,16 @@
 [
   (type_identifier)
   (unit_type)
-  "list"
 ] @type
 
+[
+  (list)
+  (list_pattern)
+] @function.builtin
+
+(list "}" @function.builtin (#set! "priority" 105))
+(list_pattern "}" @function.builtin (#set! "priority" 105))
```

Comment:
I wonder, why switching from `@type` to `@function.builtin`? I see `foo{...}` as some kind of coercion to type `foo`.

And are things can be made simpler if we match:

```
[
  ((list) "list")
  ((list_pattern) "list")
]
```

and don’t bother with the curly brackets?

Reply:
The priority is to ensure that the closing curly bracket `}` is the same color (scope) as the opening bracket.

![image](https://user-images.githubusercontent.com/16160544/188886237-bb6d1ffc-d225-4293-b70a-4b61bf5379a1.png)

I haven't found a way to just highlight `list` from node `list` and `list_pattern`

```
list_pattern [12, 4] - [12, 10]
        "list{" [12, 4] - [12, 9]
        "}" [12, 9] - [12, 10]
```

Reply:
> 
```
[
  ((list) "list")
  ((list_pattern) "list")
]
```

This query is not valid

Reply:
Ok, got it, thank you! Will try a little experiment.